tsconfig.json

```diff
@@ -10,6 +10,8 @@
       "emitDecoratorMetadata": true,
       "moduleResolution": "node",
       "importHelpers": true,
+      "lib": ["es6"],
+      "types": ["reflect-metadata"],
```

Comment:
After reading https://www.typescriptlang.org/v2/en/tsconfig#types I suggest we remove it. It narrows down what types are loaded if I understand correctly.